src/ophyd_async/tango/demo/mover.py

```diff
@@ -0,0 +1,111 @@
+import asyncio
+from dataclasses import dataclass
+from typing import Optional
+
+from bluesky.protocols import Movable, Reading, Stoppable
+
+from ophyd_async.core import (
+    DEFAULT_TIMEOUT,
+    AsyncStatus,
+    CalculatableTimeout,
+    CalculateTimeout,
+    ConfigSignal,
+    HintedSignal,
+    SignalR,
+    SignalRW,
+    SignalX,
+    WatchableAsyncStatus,
+    WatcherUpdate,
+    observe_value,
+)
+from ophyd_async.tango import (
+    TangoReadableDevice,
+)
+from tango import (
+    DevState,
+)
+
+
+@dataclass
+class TangoMoverConfig:
+    velocity: Optional[float] = None
+
+
+class TangoMover(TangoReadableDevice, Movable, Stoppable):
```

Comment:
... e.g. I could imagine
```suggestion
@tango_poll_attributes(0.1)
class TangoMover(TangoReadableDevice, Movable, Stoppable):
```
which could set `TangoMover._poll_attribute_rate = 0.1`, then `connect` could pass that information down when it creates the `TangoSignalBackend`.

What do you think?

Reply:
That seems like a good idea. I created a tango_polling class decorator that will disable events and enable polling in this way.

Reply:
tango_polling can be used as a class decorator to set either device level or signal level polling.